apps/right-to-rent-check/index.js

```diff
@@ -101,9 +101,9 @@ module.exports = {
     '/start': {
       next: '/tenant-details'
     },
-    '/current-property-address': {
+    '/tenant-current-address': {
       behaviours: AddressLookup({
-        addressKey: 'current-property-address',
+        addressKey: 'tenant-current-address',
```

Comment:
This is the change to the field name.

Reply:
yeah that was the only one and intentional as mentioned above because 1. I thought it needs to match with the url to get it to work as part of the config. 2. Keep it consistent.

I can change it back if that would help but in most of our apps the step name of any address matches the addressKey